blog/2017-01-07-monthly-release-cadence.md

```diff
@@ -0,0 +1,25 @@
+---
+title: A new Release Cadence: Releasing December and January RC
+author: Eric Vicenti
+authorTitle: Developer Advocate at Facebook
+authorURL: https://twitter.com/EricVicenti
+authorImage: https://secure.gravatar.com/avatar/077ad5372b65567fe952a99f3b627048?s=128
+authorTwitter: EricVicenti
+category: announcements
+---
+
+Shortly after React Native was introduced, we started releasing every two weeks to help the community adopt new features, while keeping versions stable for production use. At Facebook we had to stabilize the codebase every two weeks for the release of our production iOS apps, so we decided to release the open source versions at the same pace.
+
+We frequently hear feedback from the community that the release rate is hard to keep up with. These days, most of our releases are coordinated by external contributors, usually Mike Grabowski. Tools like Exponent had to skip every other release in order to manage the rapid change in version. So it seems clear that the bi-weekly releases did not serve the community well.
+
+Now, many of the Facebook apps ship once per week, especially on Android. Because we ship from master weekly, we need to keep it quite stable. So the bi-weekly release cadence doesn't even benefit internal contributors anymore.
+
+We're happy to announce the new monthly release cadence, and the new versions that have just been cut: `v0.40`(December 2016) has been stabilizing for all of December and is ready for you to use. (Just make sure to [update headers in your native modules on iOS](https://github.com/facebook/react-native/releases/tag/v0.40.0))
```

Comment:
Can you clarify here what this means regarding when releases will come out? Certain day of month?

Also, perhaps suggest that this switch will help work towards a more stable releases or other benefits I'm not seeing.

"You can now expect a new monthly release on the nth of the month. We expect the slower release cycle will allow more stability by allowing additions and features more time for the community to weigh in."